src/sentry/snuba/metrics/extraction.py

```diff
@@ -694,6 +720,9 @@ def field_to_extract(self):
         if self.op in ("on_demand_apdex"):
             return None
 
+        if self.op in ("on_demand_user_misery"):
+            return "event.user.id"
```

Comment:
This piece of code is fine but it implements the `event.` prefix automatically since that can change, I would prefer if you could just use the internal `_map_field_name` function and just give a higher-level description of the field and let the function perform the conversion. It's the same function used for all values that Relay has to extract.

```suggestion
            return _map_field_name("user.id")
```

What do you think?